src/Umbraco.Infrastructure/Persistence/NPocoSqlExtensions.cs

```diff
@@ -29,6 +30,11 @@ public static Sql<ISqlContext> Where<TDto>(this Sql<ISqlContext> sql, Expression
             return sql.Where(s, a);
         }
 
+        public static Sql<ISqlContext> WhereClosure<TDto>(this Sql<ISqlContext> sql, Expression<Func<TDto, bool>> predicate, string? alias = null)
```

Comment:
I know we don't have them for all methods already, but I think this would benefit from an XML header comment, explaining what it's for.

Reply:
Added one.